models/user.py

```diff
@@ -13,14 +25,16 @@ class RoleType(Enum):
     ADMINISTRATOR = 'Administrator'
 
 
-class ActiveType(Enum):
+class StateType(Enum):
     """Indicates the user account's state in its lifecycle.
     """
     ACTIVATION_PENDING = 'User activation pending.'
+    PASSWORT_RESET_PENDING = 'User requested a password change'
     ACTIVE = 'User account is active.'
     DISABLED = 'User account is disabled'
 
-
+# Maybe setters should not count as public.
```

Comment:
Getter/setter methods are not really Pythonic. Consider making some attributes public to access them easily with the "."-notation.